macros/SpecName.ejs

```diff
@@ -1151,9 +1139,9 @@ var specList = {
         name : 'WebVTT: The Web Video Text Tracks Format',
         url  : 'https://w3c.github.io/webvtt/'
     },
-    'Web Workers':{ /* This draft has been merged into HTML WHATWG */
-        name : 'Web Workers',
-        url  : 'https://w3c.github.io/workers/'
+    'Web Workers':{
```

Comment:
If we remove the 'Web Workers' call from these pages, we can remove this entry altogether and only use 'HTML WHATWG' going forward.

https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers
https://developer.mozilla.org/en-US/docs/Web/API/Worker
https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope
https://developer.mozilla.org/en-US/docs/Web/API/WorkerLocation
https://developer.mozilla.org/en-US/docs/Web/API/Worker/Worker
https://developer.mozilla.org/en-US/docs/Web/API/Worker/postMessage
https://developer.mozilla.org/en-US/docs/Web/API/Worker/terminate
https://developer.mozilla.org/en-US/docs/Web/API/Worker/onmessage
https://developer.mozilla.org/en-US/docs/Web/API/AbstractWorker/onerror
https://developer.mozilla.org/en-US/docs/Web/API/WorkerNavigator
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope
https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API
https://developer.mozilla.org/en-US/docs/Web/API/SharedWorker/port
https://developer.mozilla.org/en-US/docs/Web/API/SharedWorker/SharedWorker
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/navigator
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/self
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/location
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/close
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/onerror
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/onoffline
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/ononline
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/onlanguagechange
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/onclose
https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope/onmessage
https://developer.mozilla.org/en-US/docs/Web/API/SharedWorkerGlobalScope
https://developer.mozilla.org/en-US/docs/Web/API/SharedWorkerGlobalScope/applicationCache
https://developer.mozilla.org/en-US/docs/Web/API/SharedWorkerGlobalScope/name
https://developer.mozilla.org/en-US/docs/Web/API/SharedWorkerGlobalScope/onconnect
https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope/postMessage
https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope/name